fastscanner/services/indicators/tests/test_subscribe_realtime.py

```diff
@@ -127,9 +130,19 @@ async def test_prev_day_indicator(setup):
     )
     await indicator.extend(symbol, df)
 
-    channel_handler = CandleChannelHandler(symbol, [indicator], handler)
+    ts_pd = pd.Timestamp(ts)
+
+    mock_clock = MagicMock()
+    mock_clock.now.return_value = ts_pd.floor("1min") + timedelta(minutes=1, seconds=21)
```

Comment:
The bug is because the current timestamp is after the timeout. If now=21 seconds after the candle closes, now is in the future. 
You need to set `now = ts_pd.floor("1min") + pd.Timedelta(seconds=59, microseconds=990000)`. This puts the now 10ms before the candle closes.